src/main/charts/confluence/Changelog.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## TBD
+
+**Release date:** 2022-07-14
+
+![AppVersion: 7.19.0](https://img.shields.io/static/v1?label=AppVersion&message=7.19.0&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* Fix [SCALE-76](https://jira.atlassian.com/browse/SCALE-76): Fixed Confluence when Synchrony is enabled (#443)
```

Comment:
Minor, the summary for this is a bit vague i.e. "Fixed Confluence when Synchrony is enabled" might be nice to expand it a bit.